build.sbt

```diff
@@ -24,6 +26,12 @@ enablePlugins(DockerPlugin)
 
 version in Docker := "1.0"
 
+val filename = "src/main/resources/docs/patterns.json"
```

Comment:
We shouldn't use the "src/main/..." hardcoded, there is a resourceDirectory for that. This should change to something like val jsonFile = (resourceDirectory in Compile).value / "docs" / "patterns.json".

On a related note, please don't call it "filename" there are a lot of files other than the patterns.json. patternsFile, jsonFile, something that give an hint.